app/models/npc.rb

```diff
@@ -0,0 +1,51 @@
+class Npc < ActiveRecord::Base
+  class ConsequenceLimit < ActiveModel::Validator
+    def validate(npc)
+      if npc.consequences.length > 4
```

Comment:
I thought constants were to be all uppercase, though the reasoning makes sense. I suppose there are always different conventions I don't know about.